todoapp/src/components/EditTodoModal.tsx

```diff
@@ -41,34 +48,49 @@ const TodoModal = ({ todoId, handleCloseModal }: TodoModalProps) => {
   const classes = useStyles();
   const { tasksList } = useGlobalState();
   const text = useHookstate('');
+  const crudAPI = "https://crudcrud.com/api/8294360fe502453db515d4bc78dae699/todos"
```

Comment:
make a file name constants and put this contant there , so it can be useed in all over the project by exporting and also if put other constasts there too